Gems/Terrain/Code/Tests/ClipmapBoundsTests.cpp

```diff
@@ -18,7 +18,7 @@
 namespace UnitTest
 {
     class ClipmapBoundsTests
-        : public UnitTest::AllocatorsTestFixture
+        : public testing::Test
```

Comment:
Did you mean to do this?
Should this be `ScopedAllocatorSetupFixture`?

Reply:
That part is a bit of a mess right now. There's a few changes that need to be done next, but since these changes are so big already, I wanted to merge what is done before continuing.
* `ScopedAllocatorSetupFixture` needs to be removed now. The global allocator instances that are managed by `AZ::AllocatorInstance` can no longer be controlled by a scope. If you want to make one inside a scope, you do that by instantiating one of them directly, on the stack.
* `AllocatorsTestFixture` now serves the purpose of checking for memory leaks after each individual test, and nothing else. It needs to be renamed to `LeakDetectionFixture` (and will be soon). Most of the time, you want that, as it makes it easier to isolate where the leak is coming from. The problem is that for this fixture to work, it has to call `AZ::GetCurrentSerializeContextModule().Cleanup();`, so that the allocations made there don't show up as a leak. This conflicts with tests that use the `GemTestEnvironment`, which creates a ComponentApplication in environment setup and then reuse it across multiple tests.